common/p_interaction.cpp

```diff
@@ -2116,20 +2196,19 @@ void P_DamageMobj(AActor *target, AActor *inflictor, AActor *source, int damage,
 		P_KillMobj(source, target, inflictor, false);
 
 		// WDL damage events.
+		// todo: handle voodoo dolls here
 		if (source == NULL && targethasflag)
```

Comment:
Please make sure that the use of braces in conditionals is consistent.  They should either all have them or none of them have them.